tests/benchmarks/test_pauli_operator_benchmark.py

```diff
@@ -28,7 +28,7 @@
     resolve_parameter_repr,
 )
 
-N_OPERATORS_TO_BENCHMARK = 64
+N_OPERATORS_TO_BENCHMARK = 2
```

Comment:
this looks like accidentally committed

Reply:
These changes don't have to be included in this PR, but this is another one of the global variables that we should revisit after the mvp